FireboltDotNetSdk.Tests/Unit/ColumnTypeTest.cs

```diff
@@ -10,27 +10,41 @@ public void CreateColumnTypeWithArrayTypeTest()
     {
         String type = "array(integer NULL)";
         ColumnType columnType = ColumnType.Of(type);
-        Assert.That(columnType.Type, Is.EqualTo(FireboltDataType.Array));
-        Assert.False(columnType.Nullable);
-        Assert.Null(columnType.Precision);
-        Assert.Null(columnType.Scale);
-        Assert.NotNull(columnType.InnerType);
-        Assert.True(columnType.InnerType!.Nullable);
-        Assert.That(columnType.InnerType.Type, Is.EqualTo(FireboltDataType.Int));
+        if (columnType is ArrayType arrayType)
+        {
+            Assert.That(arrayType.Type, Is.EqualTo(FireboltDataType.Array));
+            Assert.False(arrayType.Nullable);
+            Assert.Null(arrayType.Precision);
+            Assert.Null(arrayType.Scale);
+            Assert.NotNull(arrayType.InnerType);
+            Assert.True(arrayType.InnerType!.Nullable);
+            Assert.That(arrayType.InnerType.Type, Is.EqualTo(FireboltDataType.Int));
+        }
+        else
+        {
+            Assert.Fail("ArrayType expected");
+        }
     }
 
     [Test]
     public void CreateColumnTypeWithNullableArrayTypeTest()
     {
         String type = "array(integer NULL) NULL";
         ColumnType columnType = ColumnType.Of(type);
-        Assert.That(columnType.Type, Is.EqualTo(FireboltDataType.Array));
-        Assert.True(columnType.Nullable);
-        Assert.Null(columnType.Precision);
-        Assert.Null(columnType.Scale);
-        Assert.NotNull(columnType.InnerType);
-        Assert.True(columnType.InnerType!.Nullable);
-        Assert.That(columnType.InnerType.Type, Is.EqualTo(FireboltDataType.Int));
+        if (columnType is ArrayType arrayType)
+        {
```

Comment:
Do we need everything wrapped in if here? It makes it a bit harder to read than it was before and conditionals in tests are a slippery slope.
Would this work?
```
if not(columnType is ArrayType arrayType) {
            Assert.Fail("ArrayType expected");
}
...
```
This way we keep it like an assert.

Reply:
But then we would still need an else clause, since the scope of `arrayType` is this if statement